packages/material-ui/src/utils/reactHelpers.js

```diff
@@ -22,3 +22,19 @@ export function isMuiElement(element, muiNames) {
 export function isMuiComponent(element, muiNames) {
   return muiNames.indexOf(element.muiName) !== -1;
 }
+
+/**
+ * passes {instanceRef} to {toRef}
+ *
+ * useful if you want to expose the ref of an inner component to the public api
+ * while still using this inside the component
+ * @param {ReactInstance} instanceRef
+ * @param {RefObject | RefCallback | any} toRef
+ */
+export function forwardRef(instanceRef, toRef) {
```

Comment:
Nice call, it's something I wanted to add. We can use in:
- `/packages/material-ui/src/Select/SelectInput.js`
- `/packages/material-ui/src/RootRef/RootRef.js`
- `/packages/material-ui/src/Input/Textarea.js`
- `/packages/material-ui/src/Input/Input.js`